catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/less/styles.less

```diff
@@ -10,8 +10,12 @@ table.align-middle{
 #node-operations {
   text-align: right;
   margin-top: 5px;
-  button {
-    padding: 5px;
+  a {
+    padding: 5px 10px;
+  }
+  .disabled-link{
+    color: grey;
+    pointer-events: none;
```

Comment:
$(a).click(function(event){ event.preventDefault();}); for disabled links
set cursor something that makes sense for ff and edge.